lib/index.js

```diff
@@ -277,9 +277,22 @@ const statsHandler = async function statsHandler(options) {
 /**
  * Show information about configured account.
  */
-const showHandler = async function showHandler({ config }) {
-  if (config === 'accountId') logger.info(await getAccountId());
-  if (config === 'apiKey') logger.info(await getAPIKey());
+const showHandler = async function showHandler({ config, all }) {
+  const showMap = {};
+  if (config === 'accountId' || all) {
```

Comment:
sorry. this still have explicit field names in it. i think something along the lines of
```
// mock code
const conf = await loadUserConf();
if (all) {
  // sort?
  for field in conf {
    logger.info(`${conf[field]}`);
  }
} else {
   // check field, and then
   logger.info(`${field}: ${conf[field]}`);
}
```

Reply:
I understand now